changelog_entry.yaml

```diff
@@ -0,0 +1,5 @@
+- bump: minor
+  changes:
+    changed:
+    - Worked on SCF imputation. 

```

Comment:
We usually try to stick to definitive 'what changed in the package?' changelog entries.